src/instance.js

```diff
@@ -76,6 +77,16 @@ Instance.getTypeType_ = function(type) {
   return types[type] || types.unspecified;
 };
 
+/**
+ * Get a reference to a Bigtable App Profile.
+ *
+ * @param {string} name The name of the cluster.
```

Comment:
Should this be the name of the appProfile?

Reply:
Fixed